core/pom.xml

```diff
@@ -290,6 +274,28 @@
 					</toolchains>
 				</configuration>
 			</plugin>
+			<plugin>
+				<groupId>com.diffplug.spotless</groupId>
+				<artifactId>spotless-maven-plugin</artifactId>
+				<version>${spotless.version}</version>
+				<configuration>
+					<java>
+						<eclipse>
+							<file>${jmc.config.path}/ide/eclipse/formatting/formatting.xml</file>
+							<version>4.8.0</version>
+						</eclipse>
+					</java>
+				</configuration>
+				<executions>
+					<execution>
+						<id>check</id>
+						<phase>validate</phase>
+						<goals>
+							<goal>check</goal>
+						</goals>
```

Comment:
**thought (non-blocking):** Have you ever thought about making the style checks before compilation ?

Since style is faster to fix than the whole compilation this could avoid awaiting long PR pre-submit feedback to fix these.

Reply:
That's what happens now. spotless:check is run during validate phase now. Far before compilation.

Reply:
Ah ok, it's been a while since I didn't do maven.